feature/org.wso2.carbon.identity.mgt.feature/pom.xml

```diff
@@ -74,6 +74,7 @@
                                         <include>dbscripts/identity-mgt/mysql.sql</include>
                                         <include>p2.inf</include>
                                         <include>build.properties</include>
+                                        <include>conf/UserLifeCycle.xml</include>
```

Comment:
Shall we use a lowercase file name like `user-life-cycle.xml`. All the other confs are in this format.